index.js

```diff
@@ -90,8 +93,42 @@ function handleClose (writeStream) {
       writeStream.emit('close')
     })
   }
+  function trapWindowsEPERMRename (err) {
+    if (err.syscall && err.syscall === 'rename' &&
+        err.code && err.code === 'EPERM' &&
+        writeStream.__isWin
+    ) {
+      if (fs.existsSync(writeStream.__atomicTarget)) {
+        fs.readFile(writeStream.__atomicTmp,
+          function (readErr, buf) {
+            if (readErr) return cleanup(err)
+
+            var atomicTmpMD5 = md5(buf)
+            fs.readFile(writeStream.__atomicTarget,
+              function (readErr, buf) {
+                if (readErr) return cleanup(err)
```

Comment:
maybe rename `readErr` and above to match the read operation? `readTempErr` and `readTargetErr`?